frontend/utils/image.js

```diff
@@ -27,14 +27,17 @@ export const imageUpload = async (imageFile) => {
     // https://developer.mozilla.org/en-US/docs/Web/API/Blob
     reader.onloadend = async function () {
       const fd = new FormData()
-      const hash = blake32Hash(new Uint8Array(reader.result))
-      console.debug('picture hash:', hash)
-      fd.append('hash', hash)
-      fd.append('data', file)
-      fetch(`${sbp('okTurtles.data/get', 'API_URL')}/file`, {
-        method: 'POST',
-        body: fd
-      }).then(handleFetchResult('text')).then(resolve).catch(reject)
+      const { result } = reader
+      if (result !== null) {
```

Comment:
Yes that would be safer. Should we pass `reader.error` to `reject()` while doing so ?

Reply:
I have no idea, but I see no harm in doing so if the documentation says that is a thing!

Reply:
I am no longer sure about rejecting the promise, we should make sure to not break the UI somewhere because of a regular file I/O error. Perhaps printing a warning to the console is alright for now?

Reply:
Sure